pkg/resource/deploy/source_eval_test.go

```diff
@@ -660,11 +660,11 @@ func TestDisableDefaultProviders(t *testing.T) {
 					aPkgProvider = providerARef.String()
 				}
 				// Perform some reads and invokes with explicit provider references.
-				_, _, perr := resmon.ReadResource("pkgA:m:typA", "resA", "id1", "", nil, aPkgProvider, "", "")
+				_, _, perr := resmon.ReadResource("pkgA:m:typA", "resA", "id1", "", nil, aPkgProvider, "", "", "")
 				aErrorAssert(t, perr)
-				_, _, perr = resmon.ReadResource("pkgB:m:typB", "resB", "id1", "", nil, providerBRef.String(), "", "")
+				_, _, perr = resmon.ReadResource("pkgB:m:typB", "resB", "id1", "", nil, providerBRef.String(), "", "", "")
 				assert.NoError(t, perr)
-				_, _, perr = resmon.ReadResource("pkgC:m:typC", "resC", "id1", "", nil, "", "", "")
+				_, _, perr = resmon.ReadResource("pkgC:m:typC", "resC", "id1", "", nil, "", "", "", "")
```

Comment:
Given that we've not extended these tests to deal with parameterisation, is it fair to say that we are moving away from using them in favour of lifecycle tests? If so, is there a route to removing them down the road?

Reply:
I think these tests were added to try and increase coverage numbers without much thought as to what "feature" exactly was being tested. So yes, they ought to be rewritten as targeted lifecycle tests but I'm not sure what coverage this was giving us. 
Most of it is probably just error paths, which honestly probably still worth having in lifecycle.